src/actions/sessionActions.js

```diff
@@ -1,61 +1,49 @@
 import { SubmissionError } from 'redux-form';
 import { browserHistory } from 'react-router';
-import * as types from './actionTypes';
+import { sessionService } from 'redux-react-session';
 import sessionApi from '../api/sessionApi';
-import * as session from '../services/sessionService';
 import { routes } from '../constants/routesPaths';
 
-export const loginSuccess = () => {
-  return { type: types.LOGIN_SUCCESS };
-};
-
-export const logoutSuccess = () => {
-  return { type: types.LOGOUT_SUCCESS };
-};
-
-export const getSessionSuccess = () => {
-  return { type: types.GET_SESSION_SUCCESS };
-};
-
-export const getSessionError = () => {
-  return { type: types.GET_SESSION_ERROR };
-};
-
 export const login = (user) => {
-  return (dispatch) => {
+  return () => {
     return sessionApi.login({ user }).then(response => {
-      session.saveUser(response.data)
-      .then(() => {
-        dispatch(loginSuccess());
-        browserHistory.push(routes.home);
+      const { token } = response;
+      sessionService.saveSession({ token }).then(() => {
+        const { image, name, userId, email } = response;
+        sessionService.saveUser({ image, name, userId, email })
+        .then(() => {
+          browserHistory.push('/home');
```

Comment:
You used `routes.login` which you imported from `routesPaths` below. Declare all routes as constants. If you have any that's dynamic, you can have it as a function.